codec-http2/src/test/java/io/netty/handler/codec/http2/DefaultHttp2ConnectionEncoderTest.java

```diff
@@ -200,20 +201,6 @@ public Object answer(InvocationOnMock invocationOnMock) throws Throwable {
     }
 
     @Test
-    public void dataWriteAfterGoAwayShouldFail() throws Exception {
```

Comment:
@Scottmitch 

well... due to all the mocking I couldn't find a way to meaningfully test goaway logic for DATA and HEADER frames as there is no explicit checks for GOAWAY in those methods, but instead we do everything via stream states.